api/resources.js

```diff
@@ -3,9 +3,16 @@ var couchapp = require('couchapp')
   , path = require('path')
   ;
 
-ddoc =  { _id:'_design/bell' }
+endpoint =  { _id:'_design/bell' }
```

Comment:
This is the most basic example of checking whether or not the user has permission to the perform the operation, "is the user logged in?"  If they are not logged in, we send an `HTTP 404` response else we execute the callback which allows the operation to complete [here](https://github.com/rjsteinert/BeLL-Apps/commit/7162f85cbd34ce39847d202fed42a2213f4e32e2#diff-0364f57fbff2fabbe941ed20c328ef1aR58).